packages/terra-form-select/src/shared/_Option.module.scss

```diff
@@ -77,6 +77,8 @@
 
   .is-active {
     background-color: var(--terra-form-select-option-active-background-color, #ebf6fd);
+    outline: var(--terra-form-select-option-active-color-outline, 2px dashed #000);
+    outline-offset: var(--terra-form-select-option-active-color-outline-offset, -2px);
```

Comment:
this variable does not have references in other themes. do we need this variable..?

Reply:
Yes, outline-offset value is same for remaining themes, hence i haven't added the outline-offset variable in other themes.

Reply:
then there is no use of the variable !!

Reply:
Removed